### ISSUE TITLE
Unusually High Results

### ISSUE THREAD

**Issue by LeeHoang2710** [feifeiobama/PatchKD#3]
![image](https://github.com/user-attachments/assets/f3741035-631a-43c8-85e3-8edf8273e0b2)
![image](https://github.com/user-attachments/assets/70da6d0a-91b4-4c6d-b1a6-891242f666d3)
![image](https://github.com/user-attachments/assets/858186c6-c087-4a6e-bbde-1f3561d7e04d)
![image](https://github.com/user-attachments/assets/1f0b77bc-760d-4110-8ac4-e52d4bc225b6)

Is it abnormal that I achieved extremely high results after training for only a few epochs? The learning rates for both the TaskNet and PatchNet are set to 0.0035. Currently, I am training only on the Market1501 dataset. Do you think there might be an issue somewhere?
![image](https://github.com/user-attachments/assets/1714c54b-8316-46af-b2de-e57a3609658d)
This is my mAP and rank1 results.  Thanks for reading :>

**Comment by feifeiobama** [comment]
I recall there is a figure in the paper that shows the forgetting on the first task. Maybe you can compare with that figure (I do not have laptop with me at the moment).

**Comment by LeeHoang2710** [comment]
> I recall there is a figure in the paper that shows the forgetting on the first task. Maybe you can compare with that figure (I do not have laptop with me at the moment).

Thank you! I forgot this is a continual learning problem. I will add more datasets and check if the accuracy remains good across all of them.
